distribution/docker/src/docker/Dockerfile.ess-fips

```diff
@@ -159,6 +154,28 @@ RUN chmod g=u /etc/passwd && \\
 
 RUN ln -sf /etc/ssl/certs/java/cacerts /usr/share/elasticsearch/jdk/lib/security/cacerts
 
+# Convert cacerts (PKCS12) to BCFKS format using POSIX-compatible shell syntax
+RUN printf "\\n" | jdk/bin/keytool -importkeystore \
+  -srckeystore /usr/share/elasticsearch/jdk/lib/security/cacerts \
+  -srcstoretype PKCS12 \
+  -destkeystore config/cacerts.bcfks \
+  -deststorepass password \
```

Comment:
I think we will need a strong password, 14 chars of Ascii. Not sure if that is explicitly required for FIPS BCFKS but aligns with our custom keystore requirements.

Reply:
tweaked this